File: Assignment_12/program2.c
/////////////////////////////////////////////////////////////////
//
// Name :          program2.c
// Discription :   Display pattern
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          21/05/2025
//
/////////////////////////////////////////////////////////////////

/*
    Input : row = 4   col = 3

    Output : 1  2  3
             1  2  3
             1  2  3
             1  2  3
*/

#include<stdio.h>

void Pattern(int iRow , int iCol)
{
   int i = 0, j = 0;

   for(i = 1; i <= iRow; i++)
   {
        for(j = 1; j <= iCol; j++)
        {
            printf("%d\t",j);
        }
        printf("\n");
   }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of Rows :");
    scanf("%d",&iValue1);

    printf("Enter number of Column :");
    scanf("%d",&iValue2);

    Pattern(iValue1, iValue2);   //function call

    return 0;
}

File: Assignment_46/program2.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program2.java
// Discription :   use to Display pattern
// Input :         Integer
// Output :        Charecters
// Author :        Shivam Rajendra Kale
// Date :          12/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class pattern
{
    public void Display(int iRow, int iCol)
    {
        int Arr[][] = new int[iRow][iCol];
        int i = 0, j = 0;
        char ch = 'A';
        char ch1 = 'a';

        for(i = 0; i < iRow; i++)
        {
            for(j = 0; j < iCol; j++,ch++,ch1++)
            {
                if((i % 2) != 0)
                {
                    System.out.print(ch1+"\t");
                }
                else
                {
                    System.out.print(ch+"\t");
                }
            }
            System.out.println();
            ch = 'A';
            ch1 = 'a';
        }
    }
}

class program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter number of columns :");
        int iCol = sobj.nextInt();

        pattern pobj = new pattern();

        pobj.Display(iRow,iCol);

        sobj.close();
    }
}